### PR TITLE
#659206-Create command content v2

### PR DESCRIPTION
**Pull request by EmilsonBlip**
Check which version of the content assistant to call

### CODE REVIEW

**Review by c410s — APPROVED** [review]

File: src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using Esprima.Ast;
+
+namespace Take.Blip.Builder.Actions.ProcessContentAssistant
+{
+    /// <summary>
+    /// Content Assistant action
+    /// </summary>
+    public static class ProcessContentAssistantExtencion
```
Comment (danteveloso) [inline src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs:17]: Change the name of this class to "ProcessContentAssistantExtension"

File: src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        public static Command CommandAnalysis(this AnalysisRequest contentAssistantResource)
+            => new Command()
+            {
+                Resource = contentAssistantResource,
+                Uri = "/assistant/analysis",
+                To = "postmaster@ai.msging.net"
+            };
+
+        /// <summary>
+        /// Serialeze result v1
```
Comment (danteveloso) [inline src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs:33]: Change to Serialize Result v1

File: src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+            {
+                HasCombination = contentResult?.Result?.Content != null,
+                Value = contentResult?.Result?.Content?.ToString() ?? string.Empty,
+                Intent = bestCombinationFound?.Intent ?? string.Empty,
+                Entities = bestCombinationFound?.Entities.ToList() ?? new List<string>()
+            });
+        }
+
+        /// <summary>
+        /// Serialeze result v1
```
Comment (danteveloso) [inline src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs:51]: The same for this

File: src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+        [DataMember(Name = "combinations")]
+        public ContentCombination[] Combinations { get; set; }
+
+        /// <summary>
+        ///
+        /// </summary>
+        [DataMember(Name = "results")]
+        public Message[] Results { get; set; }
+    }
+}
```
Comment (danteveloso) [inline src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs:101]: I think you could fill these empty summaries and remove the constructor summary

**Review by danteveloso — CHANGES_REQUESTED** [review]

File: src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs (hunk 4 above)
Comment (EmilsonBlip) [reply src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs]: True

File: src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs (hunk 3 above)
Comment (EmilsonBlip) [reply src/Take.Blip.Builder/Actions/ProcessContentAssistant/ProcessContentAssistantExtencion.cs]: true

**Review by c410s — APPROVED** [review]

**Review by c410s — APPROVED** [review]

**Review by danteveloso — APPROVED** [review]

**Review by arturpeixot — APPROVED** [review]

**Review by Guilhermealvestake — APPROVED** [review]

**Pull request merged**